src/widgets/banner/patient-banner.component.tsx

```diff
@@ -104,7 +109,11 @@ export default function PatientBanner() {
                     <ExtensionSlot
                       extensionSlotName="patient-actions-slot"
                       key="patient-actions-slot"
-                    />
+                    >
+                      <CustomOverflowMenuItemComponent>
+                        <Extension />
+                      </CustomOverflowMenuItemComponent>
```

Comment:
thought we had issues here but checked and i see we have <CustomOverflowMenuItemComponent> existing at some point above

Reply:
> Opening Component existing at some point
I didn't get it, can you please elaborate on this?

Reply:
> Opening Component existing at some point
I didn't get it, can you please elaborate on this?

Reply:
i was looking for <CustomOverflowMenuItemComponent> that tallies with the </CustomOverflowMenuItemComponent> around that line,good i found it

Reply:
do you have any existing screen shots for your changes

Reply:
Actually, this is already implemented. If you go to any patient profile(chart), on the patient banner, you can see `Actions` button on the top right corner. 

Before: Every Button in the Actions menu used CustomOverflowMenuItemComponent in their react file. Then I have to add Edit Patient Details in the menu and the button is to be made in the Patient Registration. 

We can't use CustomOverflowMenuItem in the patient registration app, since it is a part of widgets app(src/ui-components/custom-overflow-menu), so @bistenes suggested me to wrap every Extension in the CustomOverFlowMenuItem. I will attach the link to the conversation.

I hope I am able to clear your doubt.


![Screenshot_20210407-002001.png](https://user-images.githubusercontent.com/51502471/113763188-05b9f300-9737-11eb-8f67-cfb51c46100e.png)

Reply:
https://github.com/openmrs/openmrs-esm-patient-chart-widgets/pull/222#issuecomment-805690803

Reply:
> I hope I am able to clear your doubt.

am well now